dali/operators/python_function/jax_function.h

```diff
@@ -70,7 +70,7 @@ DLMTensorPtr SetupTensorResource(std::unique_ptr<DLDaliTensorResource<Backend>>
 }
 
 template <typename Backend>
-DLMTensorPtr AsDLTensor(Tensor<Backend> tensor) {
+DLMTensorPtr AsDLTensor(Tensor<Backend>&& tensor) {
```

Comment:
Total nitpick - for consistency with the rest of the file
```suggestion
DLMTensorPtr AsDLTensor(Tensor<Backend> &&tensor) {
```